docker/dev/nextcloud/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  nextcloud:
```

Comment:
I needed to include the `cron` override on mine as well

```
  cron:
    volumes:
      - ${APP_DIR:-./../../custom_apps}:/var/www/html/custom_apps
    networks:
      - external
```

Reply:
Can you post your full cron definition? I only have the `nextcloud` service running locally for me.

Reply:
I was referring to the [cron definition in the integration project](https://github.com/nextcloud/integration_openproject/blob/master/docker-compose.yml#L35-L45).

Reply:
I see. Makes sense, if we need cron, we should add it to this compose file. Do you know if cron is needed for our integration purposes?

Reply:
I do not know. [Briefly reading does not sound mandatory](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/background_jobs_configuration.html). But probably useful to have.

Reply:
I added a cron service as well now. Looking at my local installation, it seems to be working.